web_view_google_map/views/google_places_template.xml

```diff
@@ -4,12 +4,13 @@
         <t t-set="google_maps_api_key" t-value="request.env['ir.config_parameter'].sudo().get_param('google.api_key_geocode')"/>
         <t t-set="google_maps_lang_localization" t-value="request.env['ir.config_parameter'].sudo().get_param('google.lang_localization')"/>
         <t t-set="google_maps_region_localization" t-value="request.env['ir.config_parameter'].sudo().get_param('google.region_localization')"/>
+        <t t-set="google_maps_libraries" t-value="request.env['ir.config_parameter'].sudo().get_param('google.maps_libraries')"/>
```

Comment:
Note to self: this ICP is defined here:
https://github.com/OCA/geospatial/blob/12.0/base_google_map/data/google_maps_libraries.xml#L4

:heavy_check_mark: